src/main/java/org/littleshoot/proxy/impl/FileHttpResponse.java

```diff
@@ -0,0 +1,46 @@
+package org.littleshoot.proxy.impl;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.RandomAccessFile;
+
+import javax.annotation.Nonnull;
+
+import io.netty.handler.codec.http.DefaultHttpResponse;
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.netty.handler.codec.http.HttpVersion;
+
+/**
+ * Use this to answer with a file
+ */
+public class FileHttpResponse extends DefaultHttpResponse {
+
+    private File file;
+
+    public FileHttpResponse(HttpVersion version, HttpResponseStatus status, @Nonnull File file) {
+        super(version, status);
+        this.file = file;
+    }
+
+    private boolean isFileNotAccessible() {
```

Comment:
Isn't it simpler/ more common to have a isFileAccessible than isFileNotAccessible

Reply:
+1